dockstore-common/pom.xml

```diff
@@ -105,11 +105,194 @@
             <artifactId>jackson-annotations</artifactId>
         </dependency>
         <!-- broad - cromwell stuff -->
+
+        <dependency>
+            <groupId>org.broadinstitute</groupId>
+            <artifactId>wdl-draft3_2.12</artifactId>
+            <version>41</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.typelevel</groupId>
+                    <artifactId>cats-effect_2.12</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.typelevel</groupId>
+                    <artifactId>cats-core_2.12</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.google.guava</groupId>
+                    <artifactId>guava</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.typelevel</groupId>
+                    <artifactId>alleycats-core_2.12</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>eu.timepit</groupId>
+                    <artifactId>refined_2.12</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.commons</groupId>
+                    <artifactId>commons-text</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
+            <groupId>org.broadinstitute</groupId>
+            <artifactId>wdl-draft2_2.12</artifactId>
+            <version>41</version>
```

Comment:
If this needs to stay in sync with the version of wdl-draft3_2.12 above, re-use a property in the root pom instead.